code/controllers/configuration/entries/lua.dm

```diff
@@ -0,0 +1,2 @@
+/datum/config_entry/str_list/lua_path
+	protection = CONFIG_ENTRY_LOCKED | CONFIG_ENTRY_HIDDEN
```

Comment:
Do we need to add a config entry for that? If so, where?

Reply:
Read other comment, add it where other config files go if it needs to be added